library/src/tensor_operation_instance/gpu/pool3d_fwd/device_avg_pool3d_fwd_ndhwc_f16_instance.cpp

```diff
@@ -16,7 +16,7 @@ void add_device_pool3d_fwd_ndhwc_f16_instances(
         instances)
 {
     add_device_operation_instances(
-        instances, device_pool3d_fwd_ndhwc_instances<F16, F16, I32, F32, ReduceOpId, false>{});
+        instances, device_pool3d_fwd_ndhwc_instances<F16, F16, I32, F16, ReduceOpId, false>{});
```

Comment:
We want to have Compute data type with higher precision than Inputs.

Reply:
Adjusted the compute data type for the instances